### ISSUE TITLE
i18n directory not being copied into android asset

### ISSUE THREAD

**Issue by rpelissier** [ruddell/generator-jhipster-ionic#7]
Hello,

After running

```
gulp build
ionic build android
```

i18n is still missing into my android asset directory **platforms/android/assets/www**. 

In fact I cannot find a gulp task about i18n. 

I solved the pb by manually copying **app/i18n** directory to **www** and re-run 
`ionic build android`

**Comment by rpelissier** [comment]
In fact my "hack" is working once. After the next build, the i18n directory in **www** is becoming empty and I have to redo the copy operation.

**Comment by rpelissier** [comment]
I finally patched gulp/building.js with the following parts

```
var buildDependencies = [
  options['force-build'] ? 'linting' : 'linting-throw',
  'build-app',
  'build-templates',
  'build-assets',
  'build-i18n'
];
```
and at the very end
```
// copy i18n
gulp.task('build-i18n', ['clean'], function () {
  return gulp.src('app/i18n/**/*')
    .pipe(gulp.dest(paths.dist+'/i18n'));
});
```

**Comment by ruddell** [comment]
Thank you for this.  I'm currently traveling but plan on doing a release tomorrow or Friday to include this and the other pull requests

**Comment by ruddell** [comment]
This should be fixed in v2.1.0
